src/pages/studyView/chartHeader/ChartHeader.tsx

```diff
@@ -78,8 +79,10 @@ export class ChartHeader extends React.Component<IChartHeaderProps, {}> {
                                         const fileName = `${this.fileName}.${props.type.substring(0, 3).toLowerCase()}`;
                                         if (props.type === "PNG") {
                                             saveSvgAsPng(data, fileName, {backgroundColor:"#ffffff"});
-                                        } else if ((props.type === "PDF" || typeof data === "string") && data.length > 0) {
+                                        } else if (typeof data === "string" && data.length > 0) {
```

Comment:
what type is this?

Reply:
@alisman It is for SVG type, SVG will be serialized to string.